src/components/playlists/playlists.module.css

```diff
@@ -0,0 +1,38 @@
+.card_container {
+  display: grid;
+  grid-template-columns: 3rem 6rem auto auto;
+  grid-template-rows: auto;
+  grid-template-areas:
+    "image image image"
+    "details details options";
+  height: 18rem;
+  width: 18rem;
+}
+.thumbnail_container {
+  grid-area: image;
+}
+.thumbnail {
+  transition: 0.3s ease-in-out;
+  height: 12rem;
+  width: 100%;
+}
+.no_thumbnail {
+  height: 12rem;
+  width: 100%;
+}
+.video_details {
+  grid-area: details;
+}
+.options_btn {
+  grid-area: options;
+  position: absolute;
+  bottom: 5%;
+}
+.options {
+  position: absolute;
+  right: 5%;
+  bottom: 0px;
+  background-color: rgba(66, 65, 65, 0.9);
```

Comment:
try to use variable here

Reply:
This is one off case, so used color value directly

Reply:
oke